src/main/kotlin/fi/metatavu/timebank/api/impl/translate/VacationRequestTranslator.kt

```diff
@@ -0,0 +1,31 @@
+package fi.metatavu.timebank.api.impl.translate
+
+import fi.metatavu.timebank.api.persistence.model.VacationRequest
+import javax.enterprise.context.ApplicationScoped
+
+/**
+ * Translates VacationRequest objects
+ */
+@ApplicationScoped
+class VacationRequestTranslator: AbstractTranslator<VacationRequest, fi.metatavu.timebank.model.VacationRequest>() {
+
+    override fun translate(entity: VacationRequest): fi.metatavu.timebank.model.VacationRequest {
+        return fi.metatavu.timebank.model.VacationRequest(
+            id = entity.id,
+            person = entity.person!!,
+            startDate = entity.startDate!!,
+            endDate = entity.endDate!!,
+            days = entity.days!!,
+            message = entity.message!!,
+            type = entity.type!!,
+            projectManagerStatus = entity.projectManagerStatus!!,
+            hrManagerStatus = entity.hrManagerStatus!!,
+            createdAt = entity.createdAt!!,
+            updatedAt = entity.updatedAt!!
```

Comment:
Remove these null assertions where they are not needed.

Reply:
They are needed because of default value of null in primary constructor.